File: 250706/Time to Time/time-to-time.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split('\n');
const [a, b, c, d] = input[0].split(' ').map(Number);
// Please Write your code here.
const startTime = 60 * a + b;
const finishTime = 60 * c + d;
console.log(finishTime - startTime);

File: 250710/Time to Time/time-to-time.c
#include <stdio.h>

int A, B, C, D;

int main() {
    scanf("%d %d %d %d", &A, &B, &C, &D);
    
    // Please write your code here.
    int from_time;
    int to_time;

    from_time = 60 * A + B;
    to_time = 60 * C + D;
    printf("%d", to_time - from_time);
    return 0;
}